api/src/opentrons/protocol_api/core/engine/protocol.py

```diff
@@ -7,7 +7,13 @@
 from opentrons_shared_data.labware.dev_types import LabwareDefinition as LabwareDefDict
 from opentrons_shared_data.pipette.dev_types import PipetteNameType
 
-from opentrons.types import DeckSlotName, Location, Mount, MountType, Point
+from opentrons.types import (
+    DeckSlotName,
+    Location,
+    Mount,
+    MountType,
+    Point,
+)
```

Comment:
Can this unrelated formatting change be reverted?

```suggestion
from opentrons.types import DeckSlotName, Location, Mount, MountType, Point
```